components/script/layout_interface.rs

```diff
@@ -92,6 +92,19 @@ pub enum ReflowQueryType {
     ContentBoxesQuery(TrustedNodeAddress),
 }
 
+/// Extra information concerning the reason for reflowing.
```

Comment:
layout_interface is specifically for things shared with the layout task; the other enums are sent, while this one is not.